src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import core.basesyntax.figure.enums.Color;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int MAX_SIDE = 100;
+    public static final int DEFAULT_RADIUS = 10;
+    public static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final String[] figures =
+            new String[]{"Square", "Rectangle", "Triangle", "Circle", "Trapezoid"};
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int figureRandom = random.nextInt(figures.length);
+        Color randomColor = colorSupplier.getRandomColor();
+        int side1 = 1 + random.nextInt(MAX_SIDE);
```

Comment:
create separate method for getting `1 + random.nextInt(MAX_SIDE);` and use it for each type of figure